api-docs/data-model.md

```diff
@@ -516,4 +516,163 @@ Unit of measurement: _volt-ampere reactive_.
+Notice: The direction/axis relates to the accelerometer and not the machine, that is, the z-axis of the machine could be something different.  
+Unit of measurement: _g-force_.
+
+### Impact
+
+Impact (g-force) - could be calculated as the vector length of the three axis accelerometer values (always a positive number), but might come from other sources.  
+Unit of measurement: _g-force_.
+
+### AftertreatmentDieselParticulateFilterTimeToNextActiveRegeneration
+
+Time until the next active regeneration of the machine's diesel particulate filter.  
+Unit of measurement: _seconds_,
+
+### AftertreatmentSCRTimeSinceLastCleaningEvent
+
+The time since the last cleaning of the machine's Selective Catalytic Reduction unit (SCR).  
+Unit of measurement: _seconds_.
+
+### EngineFuelFilterDifferentialPressure
+
+Pressure drop across the fuel filter, which affects the pressure with which fuel is being delivered to the engine.  
+Unit of measurement: _kilopascal_.
+
+### EngineOilFilterDifferentialPressure
+
+Pressure drop across the oil filter.  
+Unit of measurement: _kilopascal_.
+
+### AcAverageFrequency
+
+Average of the individual phase frequencies. This is an average of actual values and not an average over time.  
+Unit of measurement: _hertz_.
+
+### StabilizersDeployed
+
+Whether stabilizing equipment of the machine is deployed or not, examples could be all outrigger legs is lowered to the point where they stabilize/levels the machine, or a pothole protection system is in place.  
+Unit of measurement: _Boolean_.
+
+### PlatformStowed
+
+Whether the platform or implement of the machine is stowed, for example, an articulated boom lift has both booms lowered and detracted.  
+This is often seen as a safety feature, since some machines are only allowed to drive or transported when the platform is in a stowed position. Other machines has a "high speed" drive which is only enabled when the platform is stowed.  
+Unit of measurement: _Boolean_.
+
+### RescueModeActive
+
+Whether the machine is operated in rescue mode or not.  
+For instance, a boom lift is turned off but (in order to rescue an operator positioned on the platform) the platform is lowered using the ground control emergency rescue mode.  
+Unit of measurement: _Boolean_.
+
+### BatteryStateOfHealthPercent
+
+The relative state of health (SOH) of the battery.   
+Actual maximum capacity of the battery (when fully charged) as a percentage of the initial (nominal) capacity.  
+Unit of measurement: _percent_.
+
+### BatteryRemainingRunTime
+
+Estimated (actual) remaining run time on battery with respect to the actual charge level and load.  
+Also known as "residual time of battery usage"  
+Unit of measurement: _seconds_.
+
+### BatteryChargeCyclesCount
+
+The total number of charge cycles the battery has undergone.  
+Unit of measurement: _count_.
+
+### BatteryCumulativeDischargedEnergy
+
+Total amount of energy (as in kilo Watt hours) delivered by the battery.  
```

Comment:
This so nitpicking, but since you have "Unit of measurement: _kWh_" in the next line the "(as in kilo Watt hours)" is not needed - I only had on the MI because the java doc does not show units and did not want people to expect the SI unit for energy which is joule

Reply:
Thank you. I missed that when copy-pasting all these descriptions.

I've also changed "kWh" to "kilowatt-hours", because we are not shortening units in other insights.

Reply:
sorry, for keeping nitpicking, but Watt is always shown with uppercase "W" (rule goes for all units named after a person)